components/home/public.vue

```diff
@@ -524,7 +524,7 @@ export default {
         }),
     },
 
-    mounted() {
```

Comment:
Mounted only gets called the first time a component is used. Activated gets called subsequent times it is used.